Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h

```diff
@@ -442,6 +442,16 @@ class CellHelper
   {
     return cell.getAbsId();
   }
+  // If this cell - trigger one?
+  static bool isTRU(const o2::emcal::Cell& cell)
+  {
+    return true; // To be checked with Markus
+  }
+
+  static bool isTRU(const o2::phos::Cell& cell)
+  {
+    return cell.getTRU();
+  }
```

Comment:
should not these methods be provided by the Cell classes? 
Do we need to wait for the reply of @mfasel ?

Reply:
For EMCAL we currently plan only propagate STU L1 TimeSums to the AOD, not TRU digits, due to noise reason. TRU digits will only be used for QC where we will probably use the cell class as well for, the resolution should be sufficient.

In the cell class we defined (DataFormats/EMCAL/include/DataFormatsEMCAL/Cell.h, L155)
``` c++
  Bool_t getTRU() const { return isChannelType(ChannelType_t::TRU); }
 ``` 

Ideally we would combine the function names even though we have different implementations, for the 2 detectors, then a lot of the code can be combine with templates.


Not yet decided whether we will reuse the same class also for the L1 timesum from the STU.

Reply:
@mfasDa thanks for reply, just to be sure: it is ok to merge this PR with `isTRU(const o2::emcal::Cell& cell)` returning always `true`?

Reply:
@peressounko can you adapt the `isTRU(const o2::emcal::Cell& cell)` ?

Reply:
Dear @mfasDa, @shahor02, I added TRU selection for EMCAL too, please have a look. Please note, that in current O2 neither PHOS nor EMCAL do not have such selection, so I left it for EMCAL as is.